views/thank_you.erb

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+     <link rel="stylesheet" href="//aui-cdn.atlassian.com/aui-adg/5.9.12/css/aui.min.css" media="all">
+    <script src="<%= $fqdn %>/atlassian-connect/all.js"></script>
+  </head>
+   <body>
+      <div class="ac-content">
+        <div class="aui-message aui-message-info">
+              Thank you for installing GitHub App Plugin for JIRA. Please return to a JIRA ticket to create a branch.
```

Comment:
whats the purpose of this? I went through the installation but never got this message -- was just left on the github page where it said that the app was successfully installed